shared_model/utils/string_builder.hpp

```diff
@@ -81,6 +81,37 @@ namespace shared_model {
         return *this;
       }
 
+      /**
+       * Appends a new named collection to string
+       * @param c - iterable collection to append using toString method
+       */
+      template <typename Collection>
+      std::enable_if_t<
+          std::is_same<
+              typename std::decay<decltype(
+                  std::declval<Collection>().begin()->toString())>::type,
+              std::string>::value,
+          PrettyStringBuilder &>
+      appendAll(Collection &&c) {
```

Comment:
Are these methods used in the pull request? If not, they can be removed and introduced later.